tests/integration/exec.bats

```diff
@@ -57,3 +57,43 @@ function teardown() {
   [[ ${lines[1]} == *"."* ]]
   [[ ${lines[2]} == *".."* ]]
 }
+
+@test "runc exec ls -la with --cwd" {
+  # run busybox detached
+  runc run -d --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox
+
+  runc exec --cwd /bin test_busybox ls -la
```

Comment:
Why not make this test use `/bin/pwd` instead -- so we can actually check that `--cwd` does what it says?